packages/ui-components/src/components/absolute-time-picker/absolute-time-picker.tsx

```diff
@@ -74,23 +90,41 @@ export class KvAbsoluteTimePicker implements IAbsoluteTimePicker, IAbsoluteTimeP
 		if (isEmpty(newValue)) {
 			this.inputMode = ERelativeTimeInputMode.Date;
 			if (this.selectedRangeDates.length === 0) {
-				this.fromInputValue = '';
-				this.toInputValue = '';
+				this.setInputValues('', '');
 				this.displayedMonth = fromDateInput(new Date());
 			}
 			return;
 		}
 
 		this.inputMode = newValue.mode;
-		this.fromInputValue = newValue.from;
-		this.toInputValue = newValue.to;
+		this.setInputValues(newValue.from, newValue.to);
 		const [from] = this.selectedRangeDates;
 		const date = newDate(from);
 		if (date.isValid()) {
 			this.displayedMonth = date;
 		}
 	}
 
+	private setDateLimits = (min: string, max: string) => {
+		this.minDate = min;
+		this.maxDate = max;
+	};
+
+	private setInputValues = (from: string, to: string) => {
```

Comment:
Should these arguments be optional and default to empty string? Just to avoid having this:
```typescript
this.setInputValues('', ''); // this.setInputValues();
this.setInputValues(inputDate.startOf('day').format(DATETIME_INPUT_MASK), ''); // this.setInputValues(inputDate.startOf('day').format(DATETIME_INPUT_MASK));
```

Reply:
Since we now have the reset values methods, the first case would not be needed.
In the second, in my opinion I believe it helps with readability keeping booth values to represent the limits (min and max) but I can change if you think its worth it 👍

Reply:
No need, was just a suggestion.